docs/commands/lint.md

```diff
@@ -2,239 +2,378 @@
-To learn more about preprocessors and rules, refer to the [custom rules](../resources/custom-rules.md) page.
+:::warning Important
+Unlike the `bundle` command, `lint` doesn't execute decorators. To learn more about preprocessors and rules, see the [custom rules](../resources/custom-rules.md) page.
 :::
 
-## Usage
+## Options
 
-```bash
-openapi lint
-openapi lint <entrypoints>...
-openapi lint [--max-problems=<n>] [--config=<path>] [--format=<value>]
-openapi lint [--generate-ignore-file] [--help]
-openapi lint --version
+Always the over-achiever, `lint` has many options which you can execute alone, or combine with others in a single command:
+
+```bash Alone
+redocly lint
+redocly lint <entrypoints>...
+redocly lint --config=<path>
+redocly lint --extends=<value>
+redocly lint --format=<value>
+redocly lint --generate-ignore-file
+redocly lint --help
+redocly lint --max-problems=<n>
+redocly lint --skip-preprocessor=<preprocessor>
+redocly lint --skip-rule=<rule>,<rule>
+redocly lint --version
 ```
 
-## Options
-
-Option                   | Type      | Required    | Default     | Description
--------------------------|:---------:|:------------:|:-----------:|------------
-`entrypoints`            | `array`   | no           | -           | Array of API definition filenames that need to be linted. See [the Entrypoints section](#entrypoints) for more options
-`--config`               | `string`  | no           | -           | Specify path to the [configuration file](#custom-configuration-file)
-`--extends`              | `array`   | no           | -           | [Extend a specific configuration](#extend-configuration) (defaults or config file settings)
-`--format`               | `string`  | no           | `codeframe` | Format for the output.<br />**Possible values:** `codeframe`, `stylish`, `json`, `checkstyle`
-`--generate-ignore-file` | `boolean` | no           | -           | [Generate ignore file](#generate-ignore-file)
-`--help`                 | `boolean` | no           | -           | Show help
-`--max-problems`         | `number`  | no           | 100         | Truncate output to display the specified [maximum number of problems](#max-problems)
-`--skip-preprocessor`    | `array`   | no           | -           | Ignore certain preprocessors. See the [Skip preprocessor or rule section](#skip-preprocessor-or-rule) below
-`--skip-rule`            | `array`   | no           | -           | Ignore certain rules. See the [Skip preprocessor or rule section](#skip-preprocessor-or-rule) below
-`--version`              | `boolean` | no           | -           | Show version number
+```bash Combined
+redocly lint [--max-problems=<n>] [--config=<path>] [--format=<value>]
+```
 
-## Examples
+| Option | Type | Description |
+|---|---|---|
+| entrypoints | array | Lets you [pass the entrypoints](#how-to-use-entrypoints-to-check-all-or-selected-definitions) (locations) of specific definition files in the command so `lint` only checks those files. |
```

Comment:
issue with: "only check those files".  Lint will also check any files pointed to by reference object.

Reply:
Would it be better if we just say...

Lets you pass the entrypoints (locations) of specific definition files in the command.

Reply:
That suggestion sounds good to me :heavy_plus_sign:

Reply:
```suggestion
| entrypoints | array | [Sets the entrypoints](#how-to-use-entrypoints-to-check-all-or-selected-definitions) (locations) of specific definition files to lint. If no entrypoints are provided, all the APIs defined in the Redocly configuration file are linted. |
```